src/pkg/selectors/sharepoint.go

```diff
@@ -406,7 +459,11 @@ func (s SharePointScope) Get(cat sharePointCategory) []string {
 // sets a value by category to the scope.  Only intended for internal use.
 func (s SharePointScope) set(cat sharePointCategory, v []string, opts ...option) SharePointScope {
 	os := []option{}
-	if cat == SharePointLibrary {
+
+	switch cat {
+	case SharePointLibrary:
+		os = append(os, pathComparator())
+	case SharePointList: // @keepers not sure if this is necessary... comparator to be omitted?
```

Comment:
You have the right idea here.  Since `Library` and `List` are folder-level scope categories, we need to mark them as requiring path comparators.
```suggestion
	case SharePointLibrary, SharePointList:
```